### ISSUE TITLE
no jetty-alpn-agent-2.0.6.jar

### ISSUE THREAD

**Issue by notyy** [typesafehub/activator-play-tls-example#57]
when running ./play run, following error message shows:

> Error opening zip file or JAR manifest missing : /Users/twer/source/scala/play-scala-tls-example/target/universal/stage/jetty-alpn-agent/jetty-alpn-agent-2.0.6.jar

should I download the jar and put it to the specified place by myself? or do I missed some step?

**Comment by marcospereira** [comment]
Hi,

I think we need to update the `play` script here: 

https://github.com/playframework/play-scala-tls-example/blob/fe36bb884da7be92b963aa0fb66646424a5ccbf7/play#L66

Because of https://github.com/playframework/playframework/pull/8243.

We can change it to be more resilient to updates like:

```bash
AGENT=$(find $DIR/target/universal/stage/ -name jetty-alpn-agent*.jar | head -1)
```
Do you mind opening a pull request to fix it?

**Comment by DavidACL** [comment]
User's issue here is likely that there is no explicit `readme` instruction to run `sbt stage` before running `./play run`, thus ../jetty-alpn-agent-2.0.6.jar wouldn't be there in any case.

**Comment by octonato** [comment]
This will be fixed with #71
